demo/cts_poc/0-cacerts-generator.sh

```diff
@@ -14,7 +14,7 @@ CUSTOM_EKU="1.3.6.1.5.5.7.3.36"
 
 echo -e "\nSetting up environment"
 if [ ! -f "venv/bin/activate" ]; then
-    python3.8 -m venv "venv"
+    python3.12 -m venv "venv"
```

Comment:
I believe the 3.8 is from the days of using old Ubuntu, and installing newer python through the deadsnakes ppa, and python3 still pointing to the older default. On Azure Linux 3.0, 3 is 3.12, which is very recent, and so I think we could probably revert to just `python3` everywhere.

That will cause a little less friction if Azure Linux updates python3, or when we update to Azure Linux 4.0, and should not create any problems.

It's a very minor thing obviously.

Reply:
That's what I initially did, but Ivar pointed out [above](https://github.com/microsoft/scitt-ccf-ledger/pull/298#discussion_r2063986070) that we want to pin to a specific Python version since we say pyscitt is compatible with certain Python versions only. If the python3 binary packaged with AZL3 changes and points to a new Python version (e.g., python 3.13), we risk losing control over which version is supported in pyscitt.